### PR TITLE
Functional tests - Add new test 'Change order status'

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Add new test 'Change order status'
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | no
| How to test?      | no need
| Possible impacts? | no

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/26801)

### CODE REVIEW

**Review by NeOMakinG — DISMISSED** [review]

**Review by Progi1984 — DISMISSED** [review]

**Review by boubkerbribri — DISMISSED** [review]

File: tests/UI/pages/BO/orders/view.js [hunk 1]
```diff
@@ -180,6 +182,53 @@ class Order extends BOBasePage {
...
    */
 
+  // Methods for order actions
+  /**
+   * Is update status button disabled
+   * @param page {Page} Browser tab
+   * @returns {Promise<boolean>}
+   */
+  isUpdateStatusButtonDisabled(page) {
+    return this.elementVisible(page, `${this.updateStatusButton},disabled`, 1000);
```
Comment (Progi1984) [inline tests/UI/pages/BO/orders/view.js]:
```suggestion
    return this.elementVisible(page, `${this.updateStatusButton},disabled`, 1000);
```

Is it really a comma ?

**Review by Progi1984 — COMMENTED** [review]
A small comment

File: tests/UI/pages/BO/orders/view.js (hunk 1 above)
Comment (nesrineabdmouleh) [reply tests/UI/pages/BO/orders/view.js]: Yes it's a selector with comma

Comment (PierreRambaud) [reply tests/UI/pages/BO/orders/view.js]: I'm not sure with it, using comma means you try to find `<disabled>` element :thinking: 
ping @NeOMakinG

Comment (boubkerbribri) [reply tests/UI/pages/BO/orders/view.js]:
```suggestion
    return this.elementVisible(page, `${this.updateStatusButton}[disabled]`, 1000);
```

Ref: https://www.w3schools.com/css/css_attribute_selectors.asp

Comment (nesrineabdmouleh) [reply tests/UI/pages/BO/orders/view.js]: Thank you @Progi1984 , @PierreRambaud and @boubkerbribri :)
I will fix it!

Comment (NeOMakinG) [reply tests/UI/pages/BO/orders/view.js]: `.selector, disabled` means that it selects `.selector` elements AND `<disabled />` elements which aren't official elements, the `[disabled]` is what you were searching for 👍

**Review by NeOMakinG — DISMISSED** [review]

File: tests/UI/pages/BO/orders/view.js (hunk 1 above)
Comment (nesrineabdmouleh) [reply tests/UI/pages/BO/orders/view.js]: Done thx :+1:

**Review by PierreRambaud — DISMISSED** [review]
You should rebase your branch with latest changes in the develop branch.
The CI is broken because of composer 2.2 which is breaking the phpunit binary. Rebase and everything will be ok for the Integration tests :D

**Review by boubkerbribri — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thank you @nesrineabdmouleh

**Pull request merged**